source/administration/config-settings.rst

```diff
@@ -2437,6 +2447,17 @@ See the `Guest Accounts documentation <https://docs.mattermost.com/deployment/gu
 | This feature's ``config.json`` setting is ``"GuestAttribute": ""`` with string input.                                                                                |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Admin Attribute
```

Comment:
Do we also add ``AdminAttribute`` config setting?

Reply:
I added both the Admin Attribute and Admin Filter in this PR - in hindsight I should have separated them into their respective PRs for clarity.

Reply:
According to the [config.json](https://github.com/mattermost/mattermost-server/blob/master/model/config.go#L1949) there are 2 settings, ``EnableAdminAttribute`` and ``AdminAttribute``, are these just combined here?

Reply:
From what I understand, yes - EnableAdminAttribute would need to be set to ``true`` in order for AdminAttribute to be used (applied). So there's no input needed to set AdminAttribute as long as EnableAdminAttribute is set to ``true``. 

@sbishel and I chatted about this but I'm tagging him for final confirmation. :)